### DISCUSSION TITLE
Anyone created app using Capacitor?

### DISCUSSION THREAD

**Discussion by Cybermb** [ionic-team/capacitor#7305] [General]
Have anyone created an app that is available in app store or google play? I am researching capacitor js and would like to see "real" production or beta apps that are available. Also would like to hear your experiences, challenges or journey in general.

**Comment by peitschie** [comment]
I have store approved apps for both the App Store and Play Store, but unfortunately none are publicly released.

In terms of experiences... there was nothing particularly difficult or tricky with this. I stayed with mostly core plugins and a few community plugins (Bluetooth, AppInsights) that I specifically chose and reviewed to ensure they were compliant with store policies.

I used [Framework7 ](https://framework7.io/) to provide a native-ish UI on both platforms from a unified codebase. Battery usage, responsiveness, etc. was all no problems. I've done real-time EEG graphs using chartjs, oauth-based authentication via MSAL and more recently Auth0.

I continue to use and recommend capacitor/cordova as a viable business approach due to the ease in getting skilled web developers, and wide range of NPM packages.

I've found I need to be careful with animations and some CSS transitions, as transparencies on lower end phones can cause graphics lag with higher res phones. But... generally speaking, Framework7 has not given me any major issues in terms of performance or things.

**Comment by MonuCarpenter** [comment]
Got to [hush.ac](https://hush.ac) and download the hush app for Android and iOS
